virtwho/virt/xen/xen.py

```diff
@@ -45,6 +45,10 @@ def __init__(self, section_name, wrapper, *args, **kwargs):
         self.add_key('server', validation_method=self._validate_server, required=True)
         self.add_key('username', validation_method=self._validate_username, required=True)
         self.add_key('password', validation_method=self._validate_unencrypted_password, required=True)
+        # strange issue with XEN API and the use of this environment variable
+        no_proxy = self.get('no_proxy', '').strip()
+        if len(no_proxy) > 0 and not no_proxy.startswith(',') and no_proxy != '*':
+            self.add_key('no_proxy', ',' + no_proxy)
```

Comment:
I don't have any xen server for testing ATM... Is this line really used, because `add_key()` method raises exception, when it is called, without argument `validation_method`). I would like to have at least one unit test for this case.

Reply:
[test-xen]
type=xen
server=10.73.5.208
username=root
password=Welcome1!
owner=admin

Reply:
closing the PR, does not work